docs/index.md

```diff
@@ -92,13 +92,13 @@ installation.  If you are on a 32-bit installation, please substitute "i386" for
 First, install the Docker client binary:
 
 ```
-curl https://get.docker.com/builds/Windows/x86_64/docker-latest > /bin/docker
+$ curl -L https://get.docker.com/builds/Windows/x86_64/docker-latest > /bin/docker
 ```
 
 Next, install the Docker Machine binary:
 
 ```
-curl https://github.com/docker/machine/releases/docker-machine_windows-amd64.exe > /bin/docker-machine
+$ curl -L https://github.com/docker/machine/releases/download/v0.2.0/docker-machine_windows-amd64.exe > /bin/docker-machine
```

Comment:
@ahmetalpbalkan Cool I will make a follow up PR should I add it to the `docker` target too?  I guess it's pretty much always `.exe` in Windows huh?